### PR TITLE
Documentation site

### PR DESCRIPTION
**Pull request by zoef-yt**

### CODE REVIEW

File: css/showcase.css [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+	max-width: 80ch;
+	width: 100%;
+	transition: all 0.5s ease;
+}
+
+.showcase-list-item {
+	margin: 0.5rem 0 0.5rem 2rem;
+	cursor: pointer;
+	transition: all 0.5s ease;
+}
```
Comment (bhushannaikaroskar) [inline css/showcase.css]: Hey zoef similar to this there are multiple places where you have used rem for anything below 10px. I can see this for margin, padding and font-size. I would suggest to use px for this. rest seems good for me.

**Review by bhushannaikaroskar — CHANGES_REQUESTED** [review]

**Review by srishtigautam21 — APPROVED** [review]
good work....

File: pages/documentation/documentation.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+	</head>
+	<body class="main-body">
+		<header class="body-header">
+			<h1 class="margin-right">UI Game</h1>
+			<a href="https://github.com/zoef-yt/ui-game"><h3>Github Repo</h3></a>
+		</header>
+
+		<main class="body-content"></main>
+
+		<footer class="body-footer-1">Made with ❤️ zoef shaikh</footer>
```
Comment (developer-anand) [inline pages/documentation/documentation.html]: You can put a div to contain both footer and apply common styling on the container. **just my opinion**.

**Review by developer-anand — COMMENTED** [review]
Looks good

File: pages/documentation/documentation.html (hunk 2 above)
Comment (zoef-yt) [reply pages/documentation/documentation.html:18]: This is an extra file that was included in the commit it will be removed in the next commit.

Comment (zoef-yt) [reply pages/documentation/documentation.html]: 2 footers are used for grids.

**Comment by j24m** [comment]
Hey, great work, looks clean and minimalistic. Good choice of font and colors.
Would just like to add one thing, when not providing with the option to change the mode, light mode should be primary.

**Comment by j24m** [comment]
Class names are well thought and the file is clean and readable, good to merge , great work💯

**Review by j24m — APPROVED** [review]
Everything look good to be merged. Great work.💯

**Review by tonymike99 — APPROVED** [review]
Amazing work, Zoef! The theme, transitions are very good!

File: css/documentation.css [hunk 3]
```diff
@@ -1,7 +1,160 @@
...
+	grid-area: header;
+}
+.heading:hover {
+	cursor: pointer;
+	color: var(--colour-primary);
+}
+.header-cta {
+	color: var(--colour-secondary);
+}
+
```
Comment (zoyatanweer) [inline css/documentation.css:17]: inconsistent spacing

**Review by zoyatanweer — APPROVED** [review]

**Comment by zoef-yt** [comment]
Thanks  you all for all the recommendations, changes and approving.

**Pull request merged**